tooling/helpers/websocket_helper.py

```diff
@@ -0,0 +1,91 @@
+import asyncio
+import websockets
+from pydub import AudioSegment
+import requests
+import json
+
```

Comment:
from http import HTTPStatus
```suggestion
from http import HTTPStatus
```